README.md

```diff
@@ -42,3 +42,30 @@ Then run `npm run openshift` to deploy your app
 
 Then you can navigate to the newly exposed route, something similar to "http://nodejs-rest-http-crud-boosters.192.168.99.100.nip.io/",  this will probably be different based on your Openshift IP address
 
+#### Running on Openshift with traces enabled
+
+Log in with kubeadmin and install the collector and jaeger via operator hub.
```

Comment:
Can you provide the specific names of the operators to install?

Reply:
done, thanks